src/sentry/web/urls.py

```diff
@@ -38,12 +38,9 @@
 from sentry.auth.providers.saml2 import SAML2AcceptACSView, SAML2SLSView, SAML2MetadataView
 from sentry.web.frontend.organization_auth_settings import \
     OrganizationAuthSettingsView
-from sentry.web.frontend.organization_member_settings import \
-    OrganizationMemberSettingsView
```

Comment:
can this `OrganizationMemberSettingsView` view be deleted now also?

Reply:
ooh yeah